src/docs/styles/pages/_home.scss

```diff
@@ -9,7 +9,7 @@
     padding: space(2x) space(4x) space(8x);
 
     &.bg-gradient-assertive {
-      background: $gradient-assertive;
+      @include linear-gradient($direction: $vertical, $style: $assertive);
```

Comment:
since we can't start vars with numbers, can we find a way to pass in a value from a list of accepted values for the arg?